apps/03TicTacToe.js

```diff
@@ -24,23 +24,81 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+
+//  Your code here
+  if ((board[0][0] === 'X' && board[0][1] === 'X' && board[0][2] === 'X' ) || (board[0][0] === 'O' && board[0][1] === 'O' && board[0][2] === 'O')) {
+      console.log(playerTurn + " !!! WINS!!!    ...Resetting Game");
+
+        board = [
+          [' ', ' ', ' '],
+          [' ', ' ', ' '],
+          [' ', ' ', ' ']
+        ];
+return true;
+  } else
+  return false;
 }
 
 function verticalWin() {
-  // Your code here
+        if  ((board[0][0] === 'X' && board[1][0] === 'X' && board[2][0] === 'X' )
```

Comment:
You can simplify these `if` statements to only check 'O' when it's 'O''s turn and 'X' when it's X turn.

For help, refer to how you used a ternary operator on line 99.